src/Resource/Metadata/PropertyInterface.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LAG\AdminBundle\Resource\Metadata;
+
+interface PropertyInterface
```

Comment:
Could you describe a property interface role and function ?

Reply:
Documentation has been added